.env.sample

```diff
@@ -1 +1,4 @@
 CIVO_API_KEY=
+CIVO_REGION=STAGING
```

Comment:
Theses shouldn't get set in the main repo sample. Only internal staff have access to the staging env, any public user testing this will not be able to get this working / tested

Reply:
updated to prod endpoint,region.